### ISSUE TITLE
Adjust markup of admin stats to not use table

### ISSUE THREAD

**Issue by SMaxOwok** [ManifoldScholar/manifold#2006]
Table lacks th elements which indicate to SR users what type of information is being presented

There is no need for this list to be a table, since it doesn't have any headers.  It's just a list.
